src/main/java/com/gameList/gamelist/GameListController.java

```diff
@@ -44,7 +44,7 @@ private void loadData() {
 		Xbox xbox4 = new Xbox(1,"Halo 4", "xbox");
 		Xbox xbox5 = new Xbox(1,"Halo 5", "xbox");
 		
-		xbox_list = new ArrayList();
+		xbox_list = new ArrayList<Xbox>();
```

Comment:
Because the types are consistent, the uncertainty of generic types can be solved directly by setting the specific types of properties or method parameters in the class.